.travis.yml

```
@@ -60,37 +70,27 @@ matrix:
             - gcc-7
             - g++-7
       env: DEVITO_ARCH=gcc-7 DEVITO_OPENMP=0 DEVITO_BACKEND=ops YC_CXX=g++-7 INSTALL_TYPE=conda RUN_EXAMPLES=False MPI_INSTALL=0
-  allow_failures:
-    - os: linux
-      python: "2.7"
-      env: DEVITO_ARCH=gcc-4.9 DEVITO_OPENMP=0
-    - os: osx
-      python: "2.7"
-      env: DEVITO_ARCH=clang DEVITO_OPENMP=0
 
 addons:
   apt:
     sources:
       - ubuntu-toolchain-r-test     # For gcc 4.9, 5 and 7
 
 before_install:
-  # Setup anaconda
-  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
-      wget https://repo.continuum.io/miniconda/Miniconda2-4.5.4-Linux-x86_64.sh -O miniconda.sh;
-    else
-      wget https://repo.continuum.io/miniconda/Miniconda3-4.5.4-Linux-x86_64.sh -O miniconda.sh;
-    fi
-  - bash miniconda.sh -b -p $HOME/miniconda
-  - export PATH="$HOME/miniconda/bin:$PATH"
-  - hash -r
-  - conda config --set always_yes yes --set changeps1 no
-  - conda update -q conda
-  # Useful for debugging any issues with conda
-  - conda info -a
   - if [[ "$MPI_INSTALL" == '1' ]]; then
       sudo apt-get install -y -q mpich libmpich-dev;
       export MPICH_CC=$DEVITO_ARCH;
     fi
+  # Setup anaconda
+  - if [[ $INSTALL_TYPE == 'conda' ]]; then
+      wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+      bash miniconda.sh -b -p $HOME/miniconda;
+      export PATH="$HOME/miniconda/bin:$PATH";
```

```suggestion
      export PATH="/usr/local/miniconda/bin:$PATH";
```
Path fix for different install location